apps/services/routers.py

```diff
@@ -139,6 +139,35 @@ async def create_user_request(
     }
 
 
+@services_router.post(
+    "/{university_id}//",
+    name="create_custom_hostel_accommodation",
+    response_model=JSENDOutSchema[CreateUserRequestOut],
+    summary="Сreate custom hostel accommodation",
+    responses={200: {"description": "Successful created custom hostel accommodation"}},
```

Comment:
create, not created
why custom?
why hostel accommodation, if u create a request for hostel accommodation?

Reply:
ok